src/lib/MiradorViewer.js

```diff
@@ -25,18 +25,12 @@ class MiradorViewer {
       || createStore(getReducersFromPlugins(this.plugins), getSagasFromPlugins(this.plugins));
     this.processConfig();
 
-    const viewer = {
-      store: this.store,
```

Comment:
I think we need to return the store too, in case it wasn't provided?

Reply:
Is it enough that is available as a property on the instance?

via https://deploy-preview-3286--mirador-dev.netlify.app/__tests__/integration/mirador/
```
> miradorInstance
Object { config: {…}, plugins: [], store: {…} }

> miradorInstance.store
Object { dispatch: dispatch(), subscribe: g(e), getState: f(), replaceReducer: replaceReducer(n), liftedStore: {…}, Symbol(observable): ()
 }

​


```

Reply:
👍 ; i wasn't thinking JS.

Reply:
Yep.. I think I originally wrote this thinking in 💎